### PR TITLE
Add "DOM Overlay Extension" section

### PR DESCRIPTION
**Pull request by klausw**
Sample test and Chrome implementation of this test API: https://github.com/web-platform-tests/wpt/pull/21556

### CODE REVIEW

**Comment by alcooper91** [comment]
I think overall this seems fine; but can you please also add this to the bikeshed doc?

**Comment by alcooper91** [comment]
cc @Manishearth

**Comment by klausw** [comment]
Done, PTAL.

File: index.bs [hunk 1]
```diff
@@ -532,3 +535,17 @@ enum FakeXRRegionType {
...
+
+The DOM Overlay extensions for test API SHOULD be implemented by all user agents that implement <a href="https://immersive-web.github.io/dom-overlays/">WebXR DOM Overlay Module</a>.
+
+<script type="idl">
+partial interface FakeXRInputController {
+  void setOverlayPointerPosition(float x, float y);
+};
+</script>
+
+When <dfn method for=FakeXRInputController>setOverlayPointerPosition(x, y)</dfn> is called, it sets a position within the DOM overlay in DOM coordinates for the next [=primary action=], simulating that the user is interacting with the DOM overlay. The UA will emit a [=beforexrselect=] event at this location before generating XR select events.
```
Comment (alcooper91) [inline index.bs]: Does "for the next" imply that it should be cleared after the next select event? If so, I feel like that should be clarified.

Or is it just trying to capture the "on the next animation frame" logic from elsewhere in the spec?

Comment (klausw) [reply index.bs]: Updated, it is intended to be for the next animation frame, and also cleared after a single frame. (It's not doing this clearing yet, proposed change for that is in [this CL](https://chromium-review.googlesource.com/c/chromium/src/+/2057120/1/third_party/blink/web_tests/external/wpt/resources/chromium/webxr-test.js) It's intended to be used with select events, but that's not strictly required.

**Pull request merged**